src/inflators/video.ts

```diff
@@ -0,0 +1,41 @@
+import { create360ImageMesh, createImageMesh } from "../utils/create-image-mesh";
+import { addComponent } from "bitecs";
+import { addObject3DComponent } from "../utils/jsx-entity";
+import { ProjectionMode } from "../utils/projection-mode";
+import { MediaVideo, MediaVideoData } from "../bit-components";
+import { HubsWorld } from "../app";
+import { EntityID } from "../utils/networking-types";
+import { Texture } from "three";
+
+export const VIDEO_FLAGS = {
+  CONTROLS: 1 << 2
```

Comment:
Curious, any special reason why `1 << 2` here? For example, just `1` would look like good enough to me.

Reply:
Copy paste fail, fixed. Thanks.